src/assets/prosemirror/menu.css

```diff
@@ -0,0 +1,147 @@
+.ProseMirror-textblock-dropdown {
```

Comment:
Yeah, I think these can be dropped into the component.  It's a more obvious place to go edit them.